data/css/escola.scss

```diff
@@ -203,3 +254,52 @@ where it needs to be */
   min-height: 102px;
   background-color: #323233;
 }
+
+scrollbar {
+    trough {
+      margin: 0;
+        padding: 0px 3px;
+        background-color: transparentize(black, 1 - 0.90);
+        background-image: linear-gradient(89deg,
+                                          transparentize(white, 1 - 0.1) 0%,
+                                          transparentize(black, 1 - 0.1) 100%);
+        box-shadow: inset 1px 0px 0px 0px transparentize(black, 1 - 0.15);
+    }
+
+    slider {
+        background-color: transparentize(white, 1 - 0.5);
+    }
+}
+
+.composite {
+  .home-page {
+    .landing {
+      .background {
+        min-height: 250px;
+      }
+
+      .BannerDynamic {
+        &__logo {
+          min-height: 120px;
+          min-width: 400px;
+          font-size: 60px;
+        }
+      }
+
+      .StaticText {
+        font-size: 20px;
+        margin-top: -2.0em;
+      }
+    }
+  }
+
+  .set-page,
+  .search-page {
+    .ContentGroupContentGroup,
+    .BannerSearch {
+      &__title {
+        font-size: 30px;
+      }
+    }
+  }
+}
```

Comment:
Just asking here, but was there any plans to improve the look of the search box popup?

![virtualbox_endlessos_23_02_2017_16_19_42](https://cloud.githubusercontent.com/assets/481418/23276225/463f6d92-f9e8-11e6-93de-99b0d33a8214.png)